content/en/platform/corda/5.1/application-networks/creating/members/reregister.md

```diff
@@ -0,0 +1,214 @@
+
+This section contains the following:
+1. [Inspect Member-Provided Context]({{< relref "#inspect-member-provided-context" >}})
+2. [Re-register a Member]({{< relref "#re-register-a-member-1" >}})
+
+## Inspect Member-Provided Context
+
+Currently, updates to the member-provided context are limited to custom properties (keys with the `ext.` prefix) and endpoint
+information only. Changes to other Corda platform properties are not currently supported.
+
+You may inspect member's current member-provided context either by performing a member lookup, or by looking up its latest
+registration request. For example, to look up Alice:
+
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```bash
+curl --insecure -u admin:admin -X GET $API_URL/members/$HOLDING_ID?O=Alice
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Uri "$API_URL/membership/$HOLDING_ID?O=Alice" | ConvertTo-Json -Depth 4
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+Alternatively, to retrieve the registration request:
+
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```bash
+export REGISTRATION_ID=<registration ID>
+curl --insecure -u admin:admin -X GET $API_URL/membership/$HOLDING_ID/$REGISTRATION_ID
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+$REGISTRATION_ID = <registration ID>
+Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Uri "$API_URL/membership/$HOLDING_ID/$REGISTRATION_ID"
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+To retrieve information about key pairs (for example, key ID required for the registration context) belonging to a
+holding identity, use:
+
+```bash
+curl --insecure -u admin:admin -X GET $API_URL/keys/$HOLDING_ID
+```
+
+## Re-register a Member
+
+1. Get a member-provided context of a member who has previously registered successfully, for example:
+
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```bash
+REGISTRATION_CONTEXT='{
+  "corda.session.keys.0.id": "'$SESSION_KEY_ID'",
+  "corda.session.keys.0.signature.spec": "SHA256withECDSA",
+  "corda.ledger.keys.0.id": "'$LEDGER_KEY_ID'",
+  "corda.ledger.keys.0.signature.spec": "SHA256withECDSA",
+  "corda.endpoints.0.connectionURL": "https://'$P2P_GATEWAY_HOST':'$P2P_GATEWAY_PORT'",
+  "corda.endpoints.0.protocolVersion": "1"
+}'
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+$REGISTRATION_CONTEXT = @{
+  'corda.session.keys.0.id' =  $SESSION_KEY_ID
+  'corda.session.keys.0.signature.spec' = "SHA256withECDSA"
+  'corda.ledger.keys.0.id' = $LEDGER_KEY_ID
+  'corda.ledger.keys.0.signature.spec' = "SHA256withECDSA"
+  'corda.endpoints.0.connectionURL' = "https://$P2P_GATEWAY_HOST`:$P2P_GATEWAY_PORT"
+  'corda.endpoints.0.protocolVersion' = "1"
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+2. Add a custom property to the member-provided context. In this example, the updated context contains the new custom property with the `ext.` prefix:
+
+{{< tabs >}}
```

Comment:
Code block should be indented in line with the step.

Reply:
indented